infrastructure/ami/scripts/setup-neuron.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+echo "Step: validate-neuron-devices"
+neuron-ls
+
+echo "Step: install-ubuntu-packages"
+sudo apt-get update
+sudo apt-get -y upgrade --only-upgrade systemd
+sudo apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    cmake \
+    curl \
+    emacs \
+    git \
+    git-lfs \
+    jq \
+    software-properties-common \
+    unzip \
+    vim \
+    wget \
+    zlib1g-dev \
+    libsndfile1-dev \
+    ffmpeg \
+    g++
+
+sudo apt-get update # Update to fix broken packages
```

Comment:
Why is this needed? Aren't we doing this 10 lines above?

Reply:
It will be removed